### PR TITLE
Added roadmap to readme

### PR DESCRIPTION
**Pull request by stratisMarkou**
Added roadmap to the readme, including housekeeping notes and future directions we want to take this repo in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,15 +6,25 @@ Learn to balance baby.
...
+- Clean up pendulum.py: learning-dynamics, objective, optimisation, plotting
+
+## Example notebooks
+- Pendulum notebook
 
 ## Write derivations, including complexity
+- Moment matching
 
 ## Profiling
+- Find a profiler
```
Comment (gergely-flamich) [inline README.md]: This is not specific enough, please be a bit more specific here.

**Review by gergely-flamich — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -6,15 +6,25 @@ Learn to balance baby.
 
 # Roadmap
 
-## Clean up current code
+Rename this repo. Candidates:
+- Talos
```
Comment (gergely-flamich) [inline README.md:10]: I quite like this name.

File: README.md (hunk 1 above)
Comment (stratisMarkou) [reply README.md]: Will make change.

File: README.md (hunk 2 above)
Comment (stratisMarkou) [reply README.md:10]: Yeah dude, it's great :)

**Review by gergely-flamich — APPROVED** [review]
Very good, let's see what we found next week!

**Pull request merged**